From 69204d909d3dce6b2d85e43be095b85910550015 Mon Sep 17 00:00:00 2001
From: Frank Wunderlich <frank-w@public-files.de>
Date: Wed, 19 Nov 2025 23:03:02 +0100
Subject: [PATCH 056/116] net: phy: as21xx: fix unsigned int is never less than
 zero

drivers/net/phy/as21xxx.c:549 aeon_ipc_send_cmd() warn: unsigned 'val' is never less than zero.
drivers/net/phy/as21xxx.c:557 aeon_ipc_send_cmd() warn: unsigned 'val' is never less than zero.

Reported-by: kernel test robot <lkp@intel.com>
Reported-by: Dan Carpenter <dan.carpenter@linaro.org>
Closes: https://lore.kernel.org/r/202511151513.Hnqzn6wc-lkp@intel.com/
---
 drivers/net/phy/as21xxx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/phy/as21xxx.c b/drivers/net/phy/as21xxx.c
index 4923cf52742e..87c92cc19281 100644
--- a/drivers/net/phy/as21xxx.c
+++ b/drivers/net/phy/as21xxx.c
@@ -516,7 +516,7 @@ static int aeon_ipc_send_cmd(struct phy_device *phydev,
 {
 	bool curr_parity;
 	int ret;
-	unsigned int val;
+	int val;
 
 	/* The IPC sync by using a single parity bit.
 	 * Each CMD have alternately this bit set or clear
-- 
2.39.5

